### DISCUSSION TITLE
Using Copy command BUG

### DISCUSSION THREAD

**Discussion by drewjax** [testcontainers/testcontainers-java#6155] [Q&A]
I am trying to incorporate test containers in order to implement integration tests, and this code fails because my SQL schema uses the COPY command. This is my code and the error follows.
```
public static PostgreSQLContainer postgreSQLContainer =
            new PostgreSQLContainer<>("postgres:11.1")
                     .withDatabaseName("name")
                     .withUsername("sa")
                     .withPassword("sa").withInitScript("{path/to/sql/file");
```

`ERROR: COPY from stdin failed: COPY commands are only supported using the CopyManager API.`

I cannot use this CopyManager API, is there a fix to this or am I out of luck?

**Comment by eddumelendez** [comment]
Can you share more information? Are you using docker desktop? version? which testcontainers version? OS?
